antora-playbook-author.yml

```diff
@@ -27,10 +27,6 @@ asciidoc:
     spec_version: 0.8
     kogito_examples_url: https://github.com/kiegroup/kogito-examples.git
     quarkus_cli_url: https://quarkus.io/guides/cli-tooling
-    example_project_url: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-greeting-quarkus
```

Comment:
@hmanwani-rh I think the examples root path is common and can be needed for many docs. We can use it to create dry URLs in the respective docs. wdyt?

Reply:
Yes, we do have the root path attribute available in this file as  kogito_examples_url: https://github.com/kiegroup/kogito-examples.git, that's why removing this specific link.

Reply:
Yes, but this is for just repository URL.
I was thinking if we keep a variable eg
`kogito_quarkus_example_url : https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples` in `antora-playbook.yml`
and it can be extended in specific doc file eg 
`:example_project_url: {kogito_quarkus_example_url}/serverless-workflow-callback-quarkus`. 
I feel this could be good if we ever move the examples to a different location. wdyt?
^ @ricardozanini

Reply:
@akumar074 Sure, we can add the URL for quarkus examples. However, we will be changing the structure of the Kogito examples repository, in which the URL will be changed.
Would you suggest having such attributes after the repo structure change in Kogito examples?

Reply:
I'd rather have a `kogito_examples_base_url` pointing to whatever the base is (whether the new or the old), and specifics examples composition after. Example: `kogito_example_newsletter_url` : `{kogito_examples_base_url}/whatever/my/specific/example/are`

Reply:
please sync with https://github.com/kiegroup/kogito-docs/pull/25